app_feup/lib/redux/action_creators.dart

```diff
@@ -272,6 +275,29 @@ ThunkAction<AppState> getUserSchedule(
   };
 }
 
+ThunkAction<AppState> getRestaurantsFromFetcher(Completer<Null> action){
+  return (Store<AppState> store) async{
+    try{
+      store.dispatch(SetRestaurantsStatusAction(RequestStatus.busy));
+
+      final List<Restaurant> restaurants =
+                      await RestaurantFetcherHtml().getRestaurants(store);
+      // Updates local database according to information fetched -- Restaurants
+      final RestaurantDatabase db = RestaurantDatabase();
+      db.saveRestaurants(restaurants);
+      db.restaurants(null);
```

Comment:
Instead of passing null here, can we introduce the parameter as optional, with the null value?
https://www.w3adda.com/dart-tutorial/dart-default-parameter-values

Reply:
Yes, I though that Dart  didn't support default parameter, but that link helped me figure out it had to be between curly braces, thanks